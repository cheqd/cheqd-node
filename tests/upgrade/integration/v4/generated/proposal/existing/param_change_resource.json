{
    "messages": [
        {
            "@type": "/cheqd.resource.v2.MsgUpdateParams",
            "authority": "cheqd10d07y265gmmuvt4z0w9aw880jnsr700j5ql9az",
            "params": {
                "image": {
                    "denom": "ncheq",
                    "amount": "20000000000"
                },
                "json": {
                    "denom": "ncheq",
                    "amount": "5000000000"
                },
                "default": {
                    "denom": "ncheq",
                    "amount": "10000000000"
                },
                "burn_factor": "0.990000000000000000"
            }
        }
    ],
    "metadata": "",
    "deposit": "10000000000ncheq",
    "title": "Resource Param Change",
    "summary": "This is a simple example of a param change - double the values, 99% burn factor",
    "expedited": false
}