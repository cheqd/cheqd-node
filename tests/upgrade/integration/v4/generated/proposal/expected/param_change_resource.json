{
  "image": [
    {
      "denom": "ncheq",
      "min_amount": "20000000000",
      "max_amount": "20000000000"
    }
  ],
  "json": [
    {
      "denom": "ncheq",
      "min_amount": "5000000000",
      "max_amount": "5000000000"
    }
  ],
  "default": [
    {
      "denom": "ncheq",
      "min_amount": "10000000000",
      "max_amount": "10000000000"
    }
  ],
  "burn_factor": "0.990000000000000000"
}
